Library/Formula/snappystream.rb

```diff
@@ -14,9 +14,10 @@ class Snappystream < Formula
 
   depends_on "cmake" => :build
   depends_on "snappy"
+  depends_on "boost"
```

Comment:
Is this a build-time or run-time dependency? It's an annoying big one if run-time :sob: